### ISSUE TITLE
Email: Comment: WCAG 3.0 accessibility guidelines: Accessing a "virtual" meeting using a teleconferencing application such as Zoom - receipt of streamed text content

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#476]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Mar/0003.html): 

From: Rod Macdonald (rjmacdonald@hawaiiantel.net)

Re: W3C Accessibility Guidelines (WCAG) 3.0 - W3C First Public Working Draft 21 January 2021

Issue: Accessibility to Web Content Requiring a CAPCHA for Access

Background: For purposes of this discussion, individuals who are Deaf-Blind can be grouped into four sub-groups:

(1) The Deaf-Blind individual retains sufficient residual hearing, with amplification and/or other enhancements, to access web content in the of a hearing person.

(2) The Deaf-Blind individual retains sufficient residual vision, with screen magnification and/or other enhancements, to access web content in the manner of a sighted person.

(3) The Deaf-Blind individual cannot access web content via speech or hearing, but can do so using braille.

(4) The Deaf-Blind individual cannot access web content using vision, hearing or braille, and thus cannot access web content at all. (There may be extremely rare cases when the use of unusual technology may circumvent this.)

This discussion refers exclusively to Deaf-Blind individuals in the third group - braille users.

Problem: Numerous providers of online services use a technique called a "CAPCHA" (Google calls theirs a "RECAPCHA") to ensure that customers are in fact human. Highly stylized characters are displayed on the screen and the user is asked to type the characters on the keyboard. Vision is required to do this. Typically, an audio work-around is provided so that blind individuals can gain access. However, there is no known workaround if the consumer does not have the vision OR the hearing to gain access. A consumer using a screen reader and braille display is completely excluded. 

Comments: There is no known solution to this accessibility issue.

**Comment by RealJoshue108** [comment]
@jspellman This will discussed in RQTF but my own thoughts on the issue and how it may relate to the RTC Accessibility User Requirements document is that it (the need to control and manage various output) is referred to as a [broad user need in Routing and communication channel control](https://www.w3.org/TR/raur/#routing-and-communication-channel-control)

This is where we have: User Need 5: A blind user of both screen reader and braille output devices simultaneously may need to manage audio and text output differently.

We have a requirement (REQ 5b: Allow controlled routing of alerts and other browser output to a braille device or other hardware.) that does move in parallel with the OP, and what they are suggesting. However, there is a broader issue of semantics, and user agent support that you need to be aware of.

There is a known lack of support for ARIA amongst existing braille devices. We have been discussing this in RQTF, and will be flagging this to related groups. We have discussed the possibility that in future versions of ARIA there may be a need for specific semantics that support the user need outlined here by the OP (and in the RAUR), as well as other aspects of ARIA such as Live Regions - which could be mapped to particular zones of the braille output so the user can distinguish when alerts have been fired.

So part of this is a user agent issue, in that the user agent should be able to monitor/manage the rate of output in a suitable way - in tandem with the necessary semantics.

**Comment by jasonjgw** [comment]
Upon checking the current RTC Accessibility User Requirements draft, I notice

1. I can't find a clear statement of the need that users who are deaf-blind have for text-only communication. It's implicit in some places though.
2. I can't find any mention of the issue of reading rate. The user need here would be for summaries of the dialogue rather than full word-for-word transcripts. Note that a similar issue arises in relation to captions (the choice between captions as full transcripts and captions as simplified or summarized forms of the dialogue that can be easier to understand or which can be read more slowly).
Of course, I may have overlooked something important in reviewing the draft, as it's a little while since I've read it in full.
